dev/io.openliberty.org.jboss.resteasy.cdi.ee10/src/io/openliberty/org/jboss/resteasy/common/component/LibertyResteasyCdiExtension.java

```diff
@@ -49,57 +50,32 @@ private boolean isSessionBean(AnnotatedType<?> annotatedType)
        }
        return false;
     }
-
+    
     /**
-     * Set a default scope for each CDI bean which is a RESTful WS Resource.
-     *
-     * @param <T> type
-     * @param event event
-     * @param beanManager bean manager
-     */
-    @Override
-    public <T> void observeResources(@WithAnnotations({Path.class}) @Observes ProcessAnnotatedType<T> event, BeanManager beanManager)
-    {
-       AnnotatedType<T> annotatedType = event.getAnnotatedType();
-       Class<?> javaClass = annotatedType.getJavaClass();
-       if(!javaClass.isInterface()
-           && !isSessionBean(annotatedType)
-           && !annotatedType.isAnnotationPresent(Decorator.class))
-       {
-          LogMessages.LOGGER.debug(Messages.MESSAGES.discoveredCDIBeanJaxRsResource(annotatedType.getJavaClass().getCanonicalName()));
-          //Liberty change start
-          if (Application.class.isAssignableFrom(javaClass)) {
-              event.setAnnotatedType(wrapAnnotatedType(annotatedType, applicationScopedLiteral));
-          } /* else {
-              event.setAnnotatedType(wrapAnnotatedType(annotatedType, requestScopedLiteral));
-          }
-          */
+    * Set a default scope for each CDI bean which is a RESTful WS Resource.
+    *
+    * @param <T>         type
+    * @param event       event
+    * @param beanManager bean manager
+    */
+   public <T> void observeResources(@WithAnnotations({Path.class}) @Observes ProcessAnnotatedType<T> event,
+                                    BeanManager beanManager) {
+      AnnotatedType<T> annotatedType = event.getAnnotatedType();
+      Class<?> javaClass = annotatedType.getJavaClass();
+      if (javaClass.isInterface()
+              && !isSessionBean(annotatedType)
+              && !annotatedType.isAnnotationPresent(Decorator.class)) {
+         if (!Utils.isScopeDefined(annotatedType, beanManager)) {
+            LogMessages.LOGGER.debug(Messages.MESSAGES.discoveredCDIBeanJaxRsResource(annotatedType.getJavaClass()
+                    .getCanonicalName()));
+            //Liberty change start
+            if (Application.class.isAssignableFrom(javaClass)) {
+              event.configureAnnotatedType().add(applicationScopedLiteral);
```

Comment:
Are we always assuming a request is application scoped? I understand why we would do so for `Application` subclasses, but otherwise should we be setting 
```
event.configureAnnotatedType().add(requestScopedLiteral);
```
the same as Resteasy?

I think this change probably stems from testing we had done for EE9 and weren't happy with how  Resteasy's was assigning scope, but I don't remember enough to know if we should be request scoped some of the time.

Reply:
I've chatted with Adam about this and we've agreed that, since this is consistent with the current EE9 behavior, we will leave this in for now.  I will create another issue to investigate this further and potentially change it.

Reply:
we'll have to investigate before we release, once it's released we can't change it.